packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/App.kt

```diff
@@ -85,6 +85,15 @@ public interface App {
      */
     public fun allUsers(): Map<String, User>
 
+    /**
+     * Returns a wrapper for invoking App Services Application functions as the specified [user].
```

Comment:
```suggestion
     * Returns a wrapper for invoking App Services Application functions as the specified [user].
```

I think this is just called Atlas Functions? https://www.mongodb.com/docs/atlas/app-services/functions/